Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/ScreenSpaceMousePointer.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UInput = UnityEngine.Input;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Screen Space Mouse Pointer Implementation.
```

Comment:
add description what the class does (eg keeps mouse pointer within screen boundaries)

Reply:
done

Reply:
i can't find the description. wondering if you didn't push the last commit or if github is just not showing me the change?

also if you can add one or two sentences to the spatialmousepointer class description , just what it does (mouse pointer moves around camera in 3d space , not bound to screenspace)

Reply:
Ok, I added something in. I just did not think that I should write any more than there was before I even touched the class.

Reply:
Somehow I screwed up. I have the same repository twice with just the s in upper and lower case. But I can't open both at the same time for a merge, so I have to see what I can do about that.

Reply:
Managed to merge. Now the description is there

Reply:
I know the documentation in most classes is not great - but we're trying to improve that. So whenever a new feature is added or an existing feature gets fixed / refactored we should update / improve documentation as well :)